modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+[id="update-service-install-web-console"]
+= Installing the {product-title} Update Service using the web console
+include::modules/common-attributes.adoc[]
+
+You can use the {product-title} web console to install the {product-title} Update Service operator.
+
+== Procedure
+
+To install the {product-title} Update Service operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose  *{product-title} Update Service* from the list of available Operators, and click *Install*.
+
+. FIXME: Does this apply? Ensure that the *All namespaces on the cluster* is selected under *Installation Mode*.
+
+. Select a namespace for *Installed Namespace*, such as the default FIXME-default-namespace.
+
+. FIXME: Does this apply? Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+
+. Select *{product-version}* as the *Update Channel*.
+
+. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
```

Comment:
```suggestion
** The *Manual* strategy requires a user with ~appropriate~ <whatever the right ones are> credentials to approve the Operator update.
```

Reply:
Did the `**` with adf5ca0732 -> 2c77ff9ce2 .  Left a FIXME for creds.